pyrogram/filters.py

```diff
@@ -448,7 +448,8 @@ async def private_filter(_, __, m: Message):
 # endregion
 
 # region group_filter
-async def group_filter(_, __, m: Message):
+async def group_filter(_, __, u: Update):
+    m = u.message if isinstance(u, CallbackQuery) else u
```

Comment:
But what if the callback is a `via_bot` one?

Reply:
The filter fails, as it did when applied to a callback query

Reply:
sad